pennylane/transforms/qcut.py

```diff
@@ -458,18 +458,21 @@ def graph_to_tape(graph: MultiDiGraph) -> QuantumTape:
                 wire_map[original_wire] = new_wire
                 reverse_wire_map[new_wire] = original_wire
 
-        for meas in copy_meas:
-            obs = meas.obs
-            obs._wires = Wires([wire_map[w] for w in obs.wires])
-            observables.append(obs)
-
-        # We assume that each MeasurementProcess node in the graph contributes to a single
-        # expectation value of an observable, given by the tensor product over the observables of
-        # each MeasurementProcess.
-        if len(observables) > 1:
-            qml.expval(Tensor(*observables))
-        elif len(observables) == 1:
-            qml.expval(obs)
+        if copy_meas:
+            for meas in copy_meas:
```

Comment:
It would be good to have some validation of the measurements to check they are all of the same return type. E.g.,
```python
return_types = set(meas.return_type for meas in copy_meas)
if len(return_types) > 1:
    raise ValueError("Only a single return type can be used for measurement "
                               "nodes in graph_to_tape")
else:
    return_type = types.pop()
```